deerlab/dipolarkernel.py

```diff
@@ -224,65 +254,71 @@ def calckernelmatrix(t,r,method,wex,nKnots,g):
     nr = np.size(r)
     nt = np.size(t)  
     K = np.zeros((nt,nr))
-    wr = w0(g)/(r**3)  # rad s^-1
+    ωr = ω0(g)/(r**3)  # Mrad s^-1
```

Comment:
rad µs^-1 is a bit more practically intuitive.

Reply:
Done